input/fsh/SD_RecordIdentifierFetalDeath.fsh

```diff
@@ -0,0 +1,43 @@
+RuleSet: RecordIdentifierObservation (type, code, jurisdiction, jurisdictionCode, year )
+* value[x] 0..1
+* value[x] only string   // we considered shifting to integer and kept it as string.
+* valueString ^short = "{type} Record number.  Six digit number.  Leading zeroes are optional. "
+* valueString ^maxLength = 6
+* dataAbsentReason 0..1
+* dataAbsentReason from DataAbsentReason (extensible)
+* component ..*
+* component ^slicing.discriminator.type = #value
+* component ^slicing.discriminator.path = "code"
+* component ^slicing.rules = #open
+* component contains
+    {jurisdiction} 1..1 and
+    {year} 1..1
+* component[{jurisdiction}] ^short = "Record Jurisdiction"
+* component[{jurisdiction}].code 1..1
+* component[{jurisdiction}].code = {jurisdictionCode}  // Jurisdiction Code
+* component[{jurisdiction}].value[x] 1..1
+* component[{jurisdiction}].value[x] only string
+* component[{jurisdiction}].valueString from JurisdictionVS (required)
+* component[{year}] ^short = "Year of {type}"
+* component[{year}].code = {code}             // "Date of Death/Birth"
+* component[{year}].value[x] 1..1
+* component[{year}].value[x] only dateTime
+* component[{year}].value[x] ^comment = "The record year is expressed using the YYYY portion of date."
+
+
+Profile: FetalDeathRecordIdentifier
+Parent: Observation
+Id: vrdr-fetal-death-record-identifier
+Title: "Fetal Death Record Identifier"
+Description: "Fetal Death Record Identifier (Observation).  This includes the record identifier, the jurisdiction, and the birth year."
```

Comment:
Can we revise this to mirror the BirthRecordIdentifierChild' description so it reads: "Fetal Death Record Identifier (Observation). **For use in Maternal Death.** This includes the record identifier, the jurisdiction, and the birth year."